plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_create_role_ctrl.js

```diff
@@ -0,0 +1,40 @@
+samson.controller('KubernetesCreateRoleCtrl', function($scope, $state, $stateParams, kubernetesService, kubernetesRoleFactory, notificationService, $timeout) {
+
+  $scope.action = 'Create';
```

Comment:
Damn, was an experiment, but ended up passing the label for the button through the rendering of the partial.

This code should have been removed. Thanks.